bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/i18n/netatmo_fr.properties

```diff
@@ -459,7 +459,7 @@ config.refreshInterval.description = L'a fréquence d'interrogation de l'API Net
 
 conf-error-no-client-id = Impossible de se connecter à la passerelle Netatmo car aucun id client n'est disponible dans la configuration
 conf-error-no-client-secret = Impossible de se connecter à la passerelle Netatmo car aucun secret client n'est disponible dans la configuration
-conf-error-grant-needed = La configuration est incomplète, veuillez autoriser l''extension auprès de Netatmo Connect \: ''{0}''.
+conf-error-grant-needed = La configuration est incomplète, veuillez autoriser l''extension auprès de Netatmo Connect\: {0}
```

Comment:
Please revert this change, it will have to be done at Crowdin, see https://v40.openhab.org/docs/developer/utils/i18n.html#managing-translations

Reply:
I have removed this from the commit